test/services/link/index.test.js

```diff
@@ -2,10 +2,21 @@ const assert = require('assert');
 const chai = require('chai');
 const chaiHttp = require('chai-http');
 const app = require('../../../src/app');
+const { API_HOST } = require('../../../config/globals');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['API_HOST' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5687548)